website/docs/r/event_orchestration_resource.html.markdown

```diff
@@ -28,166 +23,244 @@ resource "pagerduty_event_orchestration" "my_monitor" {
+* `name` - (Required) Name of the Event Orchestration.
+* `description` - (Optional) A human-friendly description of the Event Orchestration.
+* `team` - (Optional) Reference to the team that owns the Event Orchestration. If none is specified, only admins have access.
 
-## Example of configuring a Unrouted Rules for an Orchestration
+## Attributes Reference
 
-```hcl
-resource "pagerduty_event_orchestration_unrouted" "my_monitor" {
-  event_orchestration = pagerduty_event_orchestration.my_monitor
-  sets = [
-    {
-      id = "start"
-      rules = [
-        {
-          name = "Update the summary of un-matched Critical alerts so they're easier to spot"
-          conditions = [
-            { expression = "event.severity matches 'critical'" }
-          ]
-          actions {
-            extractions = [
-              {
-                target = "event.summary"
-                template = "[Critical Unrouted] {{event.summary}}"
-              }
-            ]
-          }
-        },
-        {
-          name = "Reduce the severity of all other unrouted events"
-          conditions = []
-          actions = {
-            severity = "info"
-          }
-        }
-      ]
-    }
-  ]
-  # In this example the user has defined their own "all other unrouted events" rule with no conditions
-  # so they aren't defining global catch_all behavior for the pagerduty_event_orchestration_unrouted resource
-  # but using the catch_all would be a totally legit alternative.
-}
+The following attributes are exported:
+
+* `id` - The ID of the Event Orchestration.
+* `integrations` - Routing keys routed to this Event Orchestration.
+
+## Import
+
+EventOrchestrations can be imported using the `id`, e.g.
+
+```
+$ terraform import pagerduty_event_orchestration.main 19acac92-027a-4ea0-b06c-bbf516519601
 ```
 
-**Argument Reference:** supports same rule, set, and catch-all arguments as the [Unrouted Orchestration endpoint](https://developer.pagerduty.com/api-reference/b3A6MzU3MDU0NDE-update-the-unrouted-orchestration-for-a-global-event-orchestration) of the Public API
+# pagerduty_event_orchestration_service
+
+When integrations exist on a service, [Service Orchestrations](https://support.pagerduty.com/docs/event-orchestration#service-orchestrations) can be used to evaluate incoming events against each of its rules, beginning with the rules in the "start" set. When a matching rule is found, it can modify and enhance the event and can route the event to another set of rules within this Service Orchestration for further processing. As a prerequisite, please ensure the service is set to evaluate orchestrations using the [Service Orchestration Active status endpoint](https://developer.pagerduty.com/api-reference/855659be83d9e-update-the-service-orchestration-active-status-for-a-service)
```

Comment:
Frank confirmed that we do not want to Terraform the active status toggle for a service to evaluate through Orchestration instead of ruleset. Adding it as part of the description here, just so there is a mention of this setting somewhere.